eng/Packages.props

```diff
@@ -25,7 +25,7 @@
     <PackageReference Update="System.Configuration.ConfigurationManager" Version="4.7.0" />
     <PackageReference Update="System.Memory" Version="4.5.4" />
     <PackageReference Update="System.Reflection.Metadata" Version="1.6.0" />
-    <PackageReference Update="System.Resources.Extensions" Version="4.6.0" />
+    <PackageReference Update="System.Resources.Extensions" Version="$(SystemResourcesExtensionsPackageVersion)" />
```

Comment:
Why this change? It makes me nervous because we have to keep this in sync with the VS repo.

Reply:
This changes allows source-build to dynamically inject the version it builds to eliminate prebuilts.  This is only done when doing a full source-build build.